docs/llms.md

```diff
@@ -17,7 +17,7 @@ To unify differences between various APIs, the framework defines a common interf
 | ------------------------------------------------------------------------- | -------------------------- | --------------------------------------------- | ---------------------------------------- |
 | `WatsonX`                                                                 | ✅                         | ⚠️ (model specific template must be provided) | ❌                                       |
 | `Ollama`                                                                  | ✅                         | ✅                                            | ⚠️ (JSON only)                           |
-| `OpenAI`                                                                  | ❌                         | ✅                                            | ⚠️ (JSON schema only)                    |
+| `OpenAI / Asure OpenAI`                                                   | ❌                         | ✅                                            | ⚠️ (JSON schema only)                    |
```

Comment:
It's "Azure", and it would probably make more sense on a separate line given that it's a separate service

Reply:
It shares the same adapter(same OpenAIChatLLM class).  Only the differences are some env variables and a flag (azure: true) in the initialization.  I can make a separate line if it's better.  WDYT?

Reply:
I'd have it as a separate line, it's still technically a different provider